python/lsst/obs/base/instrument_tests.py

```diff
@@ -27,11 +27,92 @@
 
 import abc
 import dataclasses
+import pkg_resources
 
-from lsst.obs.base import Instrument
+from lsst.obs.base import Instrument, FilterDefinitionCollection, FilterDefinition
 from lsst.obs.base.gen2to3 import TranslatorFactory
+from lsst.obs.base.yamlCamera import makeCamera
 from lsst.daf.butler import Registry
 from lsst.daf.butler import RegistryConfig
+from lsst.daf.butler.core.utils import getFullTypeName
+from lsst.daf.butler.formatters.yaml import YamlFormatter
+
+from .utils import createInitialSkyWcsFromBoresight
+
+DUMMY_FILTER_DEFINITIONS = FilterDefinitionCollection(
+    FilterDefinition(physical_filter="dummy_u", band="u", lambdaEff=0),
+    FilterDefinition(physical_filter="dummy_g", band="g", lambdaEff=0),
+)
+
+
+class TestYamlFormatter(YamlFormatter):
```

Comment:
"TestYamlFormatter" sounds like a test of the yaml formatter. You've got DummyCam below; I don't like `DummyYamlFormatter` but I can't come up with anything else (`TestingHelperYamlFormatter` isn't great either), and maybe it's fine since the docstring makes it clear?

Reply:
I've changed it to `DummyCamYamlWcsFormatter`